src/kssh/config.go

```diff
@@ -242,16 +198,51 @@ func GetDefaultBotAndTeam() (string, string, error) {
 	return lcf.DefaultBotName, lcf.DefaultBotTeam, nil
 }
 
-// Get the teamname associated with the given botname
-func GetTeamFromBot(botname string) (string, error) {
-	configs, _, err := LoadConfigs()
+// LoadConfigForBot gets the Config associated with the given botName. Will
+// need to find and load all configs from KV store.
+func (b *Bot) LoadConfigForBot(botName string) (Config, error) {
```

Comment:
made more sense to return the whole config (instead of just the team name) in order to avoid another config load